src/components/SystemMessage/SystemMessageCall.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {useEffect, useState} from 'react';
+import {SystemMessageNotif} from '../SystemMessageNotification/SystemMessageNotif';
+import {getSystemMessages} from '../../api';
+import {useMount} from '../../hooks/useMount';
+import {SystemMessage} from '../../types/SystemMessage';
+
+const SystemMessageCall = () => {
```

Comment:
Why did you rename this? Rename this back to `SystemMessageList` and the filename too